HETG_CHECK/HETG_Check.py

```diff
@@ -0,0 +1,283 @@
+#! /usr/local/bin/python
```

Comment:
I think this is OK because you're grabbing the continuity, but want to double-check that *starting* a load in the middle of the belts, the check will still execute; or does it rely on the check having been run the previous week?

Reply:
Here are the results of the MAR1422A test run. MAR1122 is the Continuity load.

MAR1422 begins with an EEPERIGEE (for our purposes).

In this case we use the MAR1122 ACIS-History.txt file to get the state at the end of the MAR1122 load since MAR1122 was a Normal load.

MAR1122A ACIS-History.txt file status line: 

     (HRC-S,HETG-IN,LETG-OUT,45643,OORMPDS,CSELFMT2,ENAB)

Notice that at the end of the MAR1122, we are in the belts. 

Output of HETG_Check.py for MAR1422:


This is the Continuity Load:
     2022:072:18:38:00.000      HETG INSERTED in the Continuity load

 HETG Check for load week:  MAR1422

This is the start of the Review Load:

     2022:072:20:22:16.153      EPERIGEE HETG Status is:    IN ok
     2022:073:03:34:48.875      HETG RETRACTED
     2022:073:03:57:48.614      OORMPEN - HETG status:      OUT


     2022:075:06:44:38.530      HETG INSERTED
     2022:075:06:46:41.591      OORMPDS - HETG status:      IN ok.
     2022:075:11:51:01.482      EPERIGEE HETG Status is:    IN ok
     2022:075:17:40:18.784      HETG RETRACTED
     2022:075:17:48:41.591      OORMPEN - HETG status:      OUT
          Percent time the HETG was in for this Perigee Passage: 99.04 %


     2022:077:22:06:01.607      HETG INSERTED
     2022:077:22:08:27.668      OORMPDS - HETG status:      IN ok.
     2022:078:03:18:46.294      EPERIGEE HETG Status is:    IN ok
     2022:078:10:36:02.615      HETG RETRACTED
     2022:078:10:54:08.811      OORMPEN - HETG status:      OUT
          Percent time the HETG was in for this Perigee Passage: 97.95 %